src/playback/qml/Audacity/Playback/components/PlaybackMeterCustomisePopup.qml

```diff
@@ -105,7 +109,7 @@ StyledPopupView {
                                     text: qsTr("Default")
 
                                     onToggled: {
-                                        root.meterStyle = PlaybackMeterStyle.Default
+                                        styleChangeRequested(PlaybackMeterStyle.Default)
```

Comment:
use `root`  for such calls: It makes it easier to distinguish calls local functions and functions from other objects.